docs/src/snap/howto/security/hardening.md

```diff
@@ -0,0 +1,3429 @@
+# {{product}} hardening guide
+
+By default, {{product}} is designed to be compliant with all hardening
```

Comment:
Contradiction with the line below "However, some hardening configurations are not applied by default" . In the CIS page we are clear that we comply with most, not all and our reasons why. "Out of the box Canonical Kubernetes complies with the majority of the recommended CIS security configurations."  We must be careful what we claim to be compliant with.